types.go

```diff
@@ -6,4 +6,5 @@ type Params struct {
 	UploadURL string   `json:"upload_url"`
 	APIKey    string   `json:"api_key"`
 	Files     []string `json:"files"`
+	Checksums []string `json:"checksums"`
```

Comment:
I've been favoring singular attribute names when using the `StringSlice` but I'm not sure I can provide a good justification for using one over the other. Good docs trumps all, I guess :)